package-testing/android-relay/app/build.gradle.kts

```diff
@@ -0,0 +1,123 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
+
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.jetbrains.kotlin.android)
+    jacoco
+
+    id("com.ncorti.ktfmt.gradle") version "0.20.1"
+}
+
+apply {
+    from("./jacoco.gradle")
+}
+
+jacoco {
+    toolVersion = "0.8.12"
+    reportsDirectory = layout.buildDirectory.dir("customJacocoReportDir")
+}
+
+
+val apiKey: String = gradleLocalProperties(
+    project.rootDir,
+    providers
+).getProperty("cloud.eppo.apiKey")
+
+val testRunnerHost: String = System.getenv("TEST_RUNNER_HOST") ?: "http://10.0.2.2"
+val testRunnerPort: String = System.getenv("TEST_RUNNER_PORT")  ?: "3000"
+
+val eppoAPIHost: String = System.getenv("EPPO_API_HOST") ?: "http://10.0.2.2"
+val appoAPIPort: String = System.getenv("EPPO_API_PORT")  ?: "5000"
+
+
+android {
+    buildFeatures.buildConfig = true
+
+    namespace = "cloud.eppo.android.sdkrelay"
+    compileSdk = 34
```

Comment:
I wonder if this should be configurable so that we can matrix test this. Or is this what just the relay server is compiled to, and/or using the matrix to select the emulator will be sufficient?

Reply:
That's a good question. I think we probably want to surface this to the build matrix.

Reply:
we should matrix test using different android emulator, `compileSdk` should stay the same I think

Reply:
We might want to extract out both the compile and target SDKs. I am curious about what customers are using for their apps.
At any rate, setting up the workflows and matrix are next, so I would like to punt parameterization to the next PR

Reply:
i've never heard of parameterizing `compileSdk` and `targetSdk` on Android apps. On my minds those are static, you can't/shouldn't change that dynamically. What changes is the runtime Android SDK version, which your code needs to be able to handle (eg. backwards compatibility). I could be wrong though